src/api/helpers/map-currencies/index.ts

```diff
@@ -0,0 +1,33 @@
+import { SUPPORTED_CURRENCIES } from '../../constants';
+import sortArrayAlphabetically from '../sort-array-alphabetically';
+import { Currency } from '../../types';
+
+/**
+ * getSupportedCurrencies
+ * Get all supported currencies
+ * @param {Array} Array of all possible currencies
+ * @returns {Array} Array of currencies that EXIP supports
+ */
+export const getSupportedCurrencies = (currencies: Array<Currency>) => {
+  const supported = currencies.filter((currency) => SUPPORTED_CURRENCIES.find((currencyCode: string) => currency.isoCode === currencyCode));
```

Comment:
do we need the const or could we just return the filter directly?

Reply:
I did think about this, I think it's good to have the const so that debugging supported currencies is easier. Also if we remove the const it'd be a very long line of code, harder to read IMO